source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -905,9 +919,9 @@ else:
 for each server in topologyDescription.servers:
     if server.address != serverDescription.address:
         if server.type is RSPrimary:
-            # See note below about invalidating an old primary.
-            replace the server with a default ServerDescription of type "Unknown"
-
+            # See note below about invalidating an old primary
+            # Replace the server with a default ServerDescription of type "Unknown" and an error field
+            # with a message containing the substring "primary marked stale due to discovery of newer primary"
```

Comment:
Should this error message be added to the table above?

Reply:
Also, could we update the spec tests for this case? For example "rs/new_primary.yml" and similar cases.

Reply:
Good catch here, I had intended to change this here.

Reply:
Realizing that this is a different case than the setVersion/electionId case, so you're right about having this message be separate.

Reply:
Thanks, what about updating the tests like "rs/new_primary.yml"?